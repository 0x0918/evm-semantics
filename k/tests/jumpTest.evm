account:
-   id: 1
-   balance: 40
-   program: PUSH 40 ; PUSH 50 ; ADD ; PUSH 0; JUMP 
-   storage: 30 , 20 , 25

.Accounts

START 1
