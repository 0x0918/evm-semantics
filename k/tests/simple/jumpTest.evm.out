<T> <k> . </k> <id> 1 </id> <pc> 0 </pc> <gas> 0 </gas> <wordStack> 90 : ( 90 : .WordStack ) </wordStack> <localMem> .Map </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 40 ) 1 |-> ( PUSH 50 ) 2 |-> ADD 3 |-> ( PUSH 0 ) 4 |-> JUMP </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 40 </balance> </account> </accounts> </T>
